sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationLiveTests.cs

```diff
@@ -13,12 +13,18 @@
 
 namespace Azure.Data.AppConfiguration.Tests
 {
+    [ClientTestFixture(
+        ConfigurationClientOptions.ServiceVersion.V1_0,
+        ConfigurationClientOptions.ServiceVersion.V2023_10_01)]
```

Comment:
Why are we including both versions for testing? I believe it's sufficient to test with the latest version only, as by default, all tests will be executed against the latest version.

Reply:
We need to assert that our library works with every service version we intend to support. Based on my work in other libraries in .NET, my understanding is that we test all supported versions when running in Live mode, but only the latest in Record/Playback. @JoshLove-msft Is this guidance documented anywhere?

Reply:
Our official guidance is to test all supported versions in Live mode, and it is documented [here](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/core/Azure.Core.TestFramework#support-multi-service-version-testing). There have been some recent discussions led by @heaths to reduce the service versions that we test against in live mode, but I don't know if those discussions have led to a change in our guidance yet.

Reply:
Ohh...then I think all the SDKs are not following this guideline.